iridium-server-base/pom.xml

```diff
@@ -62,6 +62,11 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-inline</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Is this for mocking `final` methods?

Reply:
It seems like it's needed for static mocks. Without `mockito-inline` I run into the following error:

`
org.mockito.exceptions.base.MockitoException: 
The used MockMaker SubclassByteBuddyMockMaker does not support the creation of static mocks

Mockito's inline mock maker supports static mocks based on the Instrumentation API.
You can simply enable this mock mode, by placing the 'mockito-inline' artifact where you are currently using 'mockito-core'.
Note that Mockito's inline mock maker is not supported on Android.
`

As the error message suggests one should replace `mockito-core` with `mockito-inline`.

Reply:
Nice.  You taught me something new 💯